go.mod

```diff
@@ -1,8 +1,6 @@
 module github.com/hashicorp/vault
 
-go 1.12
```

Comment:
As far as I can see all the changes to `go.mod`, `go.sum` and `sdk/go.mod` are not necessary for this PR. Would you mind to exclude them from your PR?

Reply:
Sure, I can. What are the "rules" in general about PRs which bump go/dependency versions?

Reply:
PRs never bump Go version. But you're out of date from master anyways, it's already updated for Go 1.13.

Reply:
In general, we bump the go version in a separate PR. Dependency versions can be updated in feature/bugfix PRs if there is reason/advantage to do that.

Reply:
It wasn't out of date two months ago when I submitted the PR :stuck_out_tongue: 

I'll rebase with master and push up changes.